python/gui/qgsgeometryrubberband.sip

```diff
@@ -54,7 +54,7 @@ class QgsGeometryRubberBand: QgsMapCanvasItem
     ~QgsGeometryRubberBand();
 
     /** Sets geometry (takes ownership). Geometry is expected to be in map coordinates */
-    void setGeometry( QgsAbstractGeometry* geom );
+    void setGeometry( QgsAbstractGeometry* geom /Transfer/ );
```

Comment:
I think we should fix this and make it accept a QgsGeometry instead. That'll avoid any potential leaks/deleted pointer issues.